query/src/servers/http/v1/mod.rs

```diff
@@ -15,6 +15,10 @@
 pub mod block_to_json;
 #[cfg(test)]
 mod block_to_json_test;
+pub mod http_query_handlers;
```

Comment:
Why need pub mod, maybe replace it with `mod http_query_handlers; pub use ...` is better?

Reply:
thanks，I will check all “pub”

Reply:
fixed, thank you